integration/setup.go

```diff
@@ -0,0 +1,130 @@
+package integration
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/kaspanet/kaspad/app"
+	"github.com/kaspanet/kaspad/config"
+	"github.com/kaspanet/kaspad/dbaccess"
+)
+
+func setup(t *testing.T) (app1, app2, app3 *app.App, client1, client2, client3 *rpcClient, teardownFunc func()) {
```

Comment:
I think it'll be nicer to have a struct that holds `app` and `client` and `p2pAddress`